### PR TITLE
[bug 1091905, 1150243] apply fix for bug 1091905 only to headings without subtopics

### PR DESCRIPTION
**Pull request by philipp-sumo**
hackish solution with :nth-child - if there's a better way to do so, feel free to change this.

### CODE REVIEW

**Comment by mythmon** [comment]
This looks really hacking, especially `display: table-cell`. @rehandalal, any better ideas here?

File: kitsune/sumo/static/less/products.less [hunk 1]
```diff
@@ -126,14 +126,18 @@
...
     margin-left: 88px;
-    display: table-cell;
-    height: 96px;
-    vertical-align: middle;
   }
 
   &.cf {
     margin-bottom: 20px;
+    
+    &:nth-child(2){
```
Comment (rehandalal) [inline kitsune/sumo/static/less/products.less]: nit: should be a space between `)` and `{`

File: kitsune/sumo/static/less/products.less [hunk 2]
```diff
@@ -126,14 +126,18 @@
   margin-bottom: 0;
 
   &.has-image {
+    margin-top: 0px;
```
Comment (rehandalal) [inline kitsune/sumo/static/less/products.less]: nit: alphabetize

**Comment by rehandalal** [comment]
this could probably be done using flexbox instead of `display: table-cell;` but I haven't looked at the actual bug, so I don't know what we're fixing here.

I'll look at the bug and let you know if I can think of a better way of doing this.

**Comment by philipp-sumo** [comment]
rehan, is the information in the bug reports enough to understand the problem?

File: kitsune/sumo/static/less/products.less (hunk 1 above)
Comment (rehandalal) [inline kitsune/sumo/static/less/products.less]: you can get rid of `vertical-align` and use:
```css
display: flex;
height: 96px;
flex-direction: column;
justify-content: center;
```

**Comment by rehandalal** [comment]
closed in favor of #2507

**Pull request closed without merging**